### PR TITLE
SPK-159. Apply Yoti code style

### PR DESCRIPTION
**Pull request by AlexMisiulia**

### CODE REVIEW

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/EditPersonalFormViewModel.kt [hunk 1]
```diff
@@ -22,12 +22,12 @@ import kotlinx.coroutines.withContext
...
-    private val formDao: FormDao,
-    private val personalFormRepository: PersonalFormRepository,
-    private val ioDispatcher: CoroutineDispatcher,
-    viewModelScope: CoroutineScope,
-    private val imageUtils: ImageUtils,
-    private val dateUtils: DateUtils
+        private val formDao: FormDao,
+        private val personalFormRepository: PersonalFormRepository,
+        private val ioDispatcher: CoroutineDispatcher,
+        viewModelScope: CoroutineScope,
```
Comment (cquemin) [inline app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/EditPersonalFormViewModel.kt]: this is not private or protected?

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/WriteNfcViewModel.kt [hunk 2]
```diff
@@ -23,11 +23,11 @@ import javax.inject.Inject
 private const val TAG = "WriteNfcViewModel"
 
 class WriteNfcViewModel @Inject constructor(
-    private val personalFormRepository: PersonalFormRepository,
-    viewModelScope: CoroutineScope,
-    private val nfcInteractor: NfcInteractor,
-    private val cameraInteractor: CameraInteractor,
-    private val imageUtils: ImageUtils
+        private val personalFormRepository: PersonalFormRepository,
+        viewModelScope: CoroutineScope,
```
Comment (cquemin) [inline app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/WriteNfcViewModel.kt]: same question here

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/EditPersonalFormViewModel.kt (hunk 1 above)
Comment (AlexMisiulia) [reply app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/EditPersonalFormViewModel.kt]: Yes, just need it to pass to super constructor

File: app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/WriteNfcViewModel.kt (hunk 2 above)
Comment (AlexMisiulia) [reply app/src/main/java/com/yoti/mobile/android/social/keys/features/writekey/WriteNfcViewModel.kt]: Yes, just need it to pass to super constructor

**Pull request merged**